core/shared/src/main/scala/monocle/Iso.scala

```diff
@@ -336,6 +336,13 @@ object Iso {
   /** alias for [[PIso]] id when S = T and A = B */
   def id[S]: Iso[S, S] =
     PIso.id[S, S]
+
+  /**
+    * create a [[PIso]] where S = T and A = B from
```

Comment:
I don't think we need the first line of comment.

Reply:
As in, is `create a [[PIso]] where S = T and A = B` redundant? I guess we could change the whole comment block to
```
/** create a [[Iso]] from a function that is its own inverse */
```

Reply:
yep perfect. Maybe `an` [[ISO]]